content/neuvector-prime/aws/contents.lr

```diff
@@ -0,0 +1,164 @@
+title: NeuVector Prime in AWS: Usage Instructions
+---
+theme: suse
+---
+body:
+
+## Prerequisites
+
+1. These instructions assume you have an EKS cluster installed and that you have access to that cluster via a `kubeconfig`.
+2. Installation requires you have the following tools available and properly configured to access your AWS account, and your EKS cluster:
+* `aws`
+* `curl`
+* `eksctl`
+* `helm` (v3 or greater)
+
+## Preparing your cluster
+
+### OIDC provider
+
+Your EKS cluster is required to have an OIDC provider installed. To check for an OIDC provider first find the OIDC issuer with the following command. Substitute `$CLUSTER_NAME` with the *Name* of the EKS cluster:
+
+```
+aws eks describe-cluster --name $CLUSTER_NAME --query cluster.identity.oidc.issuer --output text
+```
+
+A URL is returned, like `https://oidc.eks.region.amazonaws.com/id/1234567890ABCDEF`. The part after `https://` will be referred to in later instructions as the *OIDC Provider Identity*. The final section of the URL, `1234567890ABCDEF`, is the $OIDC_ID.
+
+Using the id of the issuer found above you can check if a provider is installed with the following command:
+
+```
+aws iam list-open-id-connect-providers | grep $OIDC_ID
+```
+
+If there is no output, you will need to create an OIDC provider:
+
+```
+eksctl utils associate-iam-oidc-provider --cluster $CLUSTER_NAME --approve
+```
+
+This will return the OIDC provider's ARN, with the ID at the end.
+
+### IAM Role
+
+To provide the necessary permissions, an IAM policy and an attached role are required. The role is passed as an argument during the *helm* deployment. Download [`neuvector-csp-iam-policy.json`](https://suse-marketplace-assets-public.s3.amazonaws.com/neuvector-prime/neuvector-csp-iam-policy.json) to locally inspect the policy. Create the policy with a *policy name* of your choosing:
+
```

Comment:
Is there a reason not to have the policy in git and pull it from there? How doe we prevent the S3 file from being stale?

Reply:
The currents docs refer to the S3 locations. I will submit a PR moving the supplemental documents to this repo, after we have retired the old docs.

Reply:
(I've had them in a branch for about a week now.)